src/diffpy/labpdfproc/tools.py

```diff
@@ -236,10 +237,10 @@ def load_user_info(args):
     the updated argparse Namespace with username and email inserted
 
     """
-    config = {"username": args.username, "email": args.email}
-    config = get_user_info(config)
-    args.username = config["username"]
-    args.email = config["email"]
+    check_and_build_global_config()
```

Comment:
since here we are allowing people to pass in this info as an arg, probably the workflow should be

- if the arg is set, use the arg and don't even run the config updater
- if the arg is not set, run the config updater
- run get_user_info passing in the args if they are not None.

-